_src/_assets/js/dashboard-charts.js

```diff
@@ -13,6 +13,7 @@
 
   const totalColor = '#585BC1'
   const positiveColor = '#FFA270'
+  const deathsBarColor = '#3F4856'
```

Comment:
Maybe this should just be `deathsColor` to match the other variable naming scheme?